### PR TITLE
Add Zenodo Metadata

### PR DESCRIPTION
**Pull request by ax3l**
This is a meta-data file for our software release archives on Zenodo.org

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Lehe, Remi",
+        "orcid": "0000-0002-3656-9659"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Qiang, Ji",
+        "orcid": "0000-"
```
Comment (ax3l) [inline .zenodo.json]: TODO: @qianglbl do you have an ORCiD?

**Review by WeiqunZhang — APPROVED** [review]
Mine is correct.

File: .zenodo.json (hunk 1 above)
Comment (ax3l) [reply .zenodo.json]: Will remove for now, we can add it with the next release.
```suggestion
```

**Review by cemitch99 — COMMENTED** [review]
Looks good.

**Pull request merged**